radicle-httpd/src/api/v1/projects.rs

```diff
@@ -1310,4 +1310,34 @@ mod routes {
             )
         );
     }
+
+    #[tokio::test]
+    async fn test_blob_stream() {
+        use crate::api::json::blob_stream;
+        use futures_util::StreamExt;
```

Comment:
can you move these `use` statements outside?

Reply:
will do.